networking/custom-domain.html.markerb

```diff
@@ -58,37 +91,90 @@ fly certs add example.com
   Status                      =
 ```
 
-Running `fly certs add` starts the process of getting a certificate. 
-
-Run `fly certs show` to get the details needed for your next step. For example:
+If you need a wildcard domain, put quotes around the hostname to avoid shell expansion:
 
 ```cmd
-fly certs show example.com
-```
-```output
-  Hostname                    = example.com
-  Configured                  = true
-  Issued                      = ecdsa, rsa
-  Certificate Authority       = lets_encrypt
-  DNS Provider                = enom
-  DNS Validation Instructions = CNAME _acme-challenge.example.com => example.com.5xzw.flydns.net.
-  DNS Validation Hostname     = _acme-challenge.example.com
-  DNS Validation Target       = example.com.5xzw.flydns.net
-  Source                      = fly
-  Created At                  = 1m24s ago
-  Status                      = Ready
+fly certs add "*.example.com"
 ```
 
-The **DNS Validation Instructions** are an optional next step. For a short time (minutes) after we start the process of generating
-the first-ever certificate for your site, trying to load that site with an HTTPS URL will generate errors. If you'd like to make sure
-those errors aren't ever visible, you can use a DNS challenge to pre-generate the certificate. 
+Running `fly certs add` starts the process of getting a certificate. `Configured` should be true and `Status` will change to `Ready` when the certificates are available. You can run `fly certs show <hostname>` to check the status.
+
+
+### (Optional) Validate with an ACME challenge
```

Comment:
It's a bit less elegant (but more accurate and less confusing in tickets) to say ACME DNS01 challenge instead of just ACME challenge. When troubleshooting, it's useful to distinguish between the two.